### PR TITLE
Multiple L10N fixes

### PR DESCRIPTION
**Pull request by johngian**
* Exclude /api and /admin URLs from L10N middleware.
* Fix LANGUAGES definition format.
* Use regex for URLs that are not to be localized.

### CODE REVIEW

File: mozillians/settings/base.py [hunk 1]
```diff
@@ -72,8 +72,10 @@
 LANGUAGE_CODE = 'en-US'
 LOCALE_PATHS = [path('locale')]
 EXEMPT_L10N_URLS = [
-    '/oidc/authenticate/',
-    '/oidc/callback/'
+    '^/oidc/authenticate/',
+    '^/oidc/callback/',
+    '^/api/v2/',
+    '^/admin/'
```
Comment (akatsoulas) [inline mozillians/settings/base.py]: Let's add `/api/v1` here too.

**Comment by johngian** [comment]
We should also fix LANGUAGES references that handle languages as dict.

File: mozillians/settings/base.py [hunk 2]
```diff
@@ -118,11 +120,11 @@ def get_langs():
 def lazy_langs():
     from product_details import product_details
 
-    return dict([(lang.lower(), product_details.languages[lang]['native'])
-                 for lang in get_langs() if lang in product_details.languages])
+    return [(lang.lower(), product_details.languages[lang]['native'])
```
Comment (akatsoulas) [inline mozillians/settings/base.py]: Out of curiosity, what's the reason for this change?  If this is not something relevant to this PR, please revert the change.

Comment (johngian) [inline mozillians/settings/base.py]: We are changing this because of the way ``Puente`` works. It tries to get closer to how django internal l10n support works.

https://docs.djangoproject.com/en/1.10/ref/settings/#languages
